addendum.dict

```diff
@@ -59,6 +60,7 @@ captchas	𐑒𐑨𐑐𐑗𐑩𐑟
 Cardew	𐑒𐑸𐑛𐑿
 Carma	𐑒𐑸𐑥𐑩
 Carmantic	𐑒𐑸𐑥𐑨𐑯𐑑𐑦𐑒
+Carnegie 𐑒𐑸𐑯𐑰𐑜𐑰
```

Comment:
I'd prefer this to be 𐑒𐑸𐑯𐑰𐑜𐑦 to comply with principle 11 at https://www.shavian.info/spelling/.